stories/index.js

```diff
@@ -213,4 +213,9 @@ storiesOf('ChipInput', module)
         fullWidth
       />
     </form>
-  ))
+  )).add('allows duplicates', () => themed(
```

Comment:
This story should better be named _allowing duplicates_ and `.add` should start on a new line.